src/bin/cql-stress-cassandra-stress/operation/user.rs

```diff
@@ -159,10 +119,11 @@ pub struct UserOperationFactory {
     pk_seed_distribution: Arc<dyn Distribution>,
     stats: Arc<ShardedStats>,
     table_metadata: Table,
-    statements_map: HashMap<String, PreparedStatement>,
+    queries_payload: HashMap<String, (PreparedStatement, f64)>,
     pk_generator_factory: Box<dyn ValueGeneratorFactory>,
     column_generator_factories: Vec<Box<dyn ValueGeneratorFactory>>,
     max_operations: Option<u64>,
+    settings: Arc<CassandraStressSettings>,
 }
```

Comment:
At least in this commit, it's not clear to me what are these `settings` and why we add them here.

Reply:
It actually was the last commit that could explain this. So why?

Reply:
> it's not clear to me what are these `settings` and why we add them here.

I just want to pass the `clustering` object from the settings which is of type `Box<dyn DistributionFactory>`. Unfortunately, we cannot clone is since it is not `Sized`. We also cannot just move the `clustering` from the settings since it's behind a shared reference (`Arc`). Do you have any idea on how to pass the distribution factory here (instead of passing whole `CassandraStressSettings` structure)?

Reply:
Maybe I could just wrap it with `Arc`, instead of `Box`. Then we could have a shared reference to the distribution factory. Let me try this out.

Reply:
> Maybe I could just wrap it with `Arc`, instead of `Box`. Then we could have a shared reference to the distribution factory. Let me try this out.

Yup, this will work. This also gets rid of the unwrap you mentioned in the other comment.